hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMultiTenantManagerImpl.java

```diff
@@ -374,10 +374,15 @@ public void deactivateUser(String accessID)
   }
 
   @Override
-  public boolean isTenantAdmin(String user, String tenantName) {
+  public boolean isTenantAdmin(String user, String tenantId) {
```

Comment:
Yes. Ideally `OMMultiTenantManager` and `OMTenantRequestHelper` should be merged. While writing the Request code earlier it was easier to extract and add my own helper methods rather than calling `OMMultiTenantManager` methods. It is due for a refactoring. @prashantpogde and I can probably work together on this.

Reply:
I filed HDDS-6387 for this. In the mean time, can we fix this particular case now since we don't want it to get lost in the merge? Probably easiest to have the one caller use the properly implemented version in OMTenantRequestHelper and make this stub return false for now to be safe.